core/java/test/org/openda/exchange/dataobjects/NetcdfFileConcatenaterTest.java

```diff
@@ -68,60 +68,57 @@ public void testDflowfmHisfileConcatenation() throws IOException {
 	}
 
 	public void testNetcdfFixedTimeDimensionConcatenation() {
-		File firstFile = new File(this.testRunDataDir, "toAdd.nc");
-		File targetFile = new File(this.testRunDataDir, "concatenated.nc");
+		File firstFile = new File(this.testRunDataDir, "concatenateFixedTimeDim/toAdd.nc");
```

Comment:
Hmm I see explicit file separators here and at other places as well. I think you.should use new File recursively or the file separator method. Note this happens at many places in the file.